### DISCUSSION TITLE
HistoryDepth greater than 84 ?

### DISCUSSION THREAD

**Discussion by colossus-digital** [polkascan/py-substrate-interface#290] [Q&A] [answered]
Hi, is there a way to query data older than 84 eras (the default HistoryDepth) ?

tnks
Andrea

**Comment by arjanz** [comment, marked as answer]
Then you will have to provide a block hash to `query()`, that will query the state at that specific time

**Comment by colossus-digital** [comment]
it works, thank you very much !
